data/json/items/armor.json

```diff
@@ -10210,5 +10210,112 @@
     "material_thickness": 1,
     "environmental_protection": 1,
     "flags": [ "OVERSIZE", "OUTER", "WATERPROOF", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "bigback",
+    "type": "ARMOR",
+    "name": "high-volume rucksack",
+    "description": "Also know as \"a bigpack\", this rucksack is prized by pack rats but may lead to a bad back.",
+    "weight": 3539,
+    "volume": 85,
+    "price": 20000,
+    "material": [ "steel", "leather" ],
+    "symbol": "[",
+    "color": "black",
+    "covers": [ "TORSO" ],
+    "coverage": 95,
+    "encumbrance": 60,
+    "storage": 400,
+    "warmth": 5,
+    "material_thickness": 6,
+    "flags": [ "BELTED", "WATERPROOF", "ONLY_ONE", "OVERSIZE" ]
+  },
+  {
+    "id": "backpack_hiking",
+    "type": "ARMOR",
+    "name": "hiking backpack",
+    "description": "A large sized hiking backpack with plenty of storage space.",
+    "weight": 2291,
+    "volume": 55,
+    "price": 10000,
+    "material": [ "leather" ],
+    "symbol": "[",
+    "color": "brown",
+    "covers": [ "TORSO" ],
+    "coverage": 75,
+    "encumbrance": 35,
+    "storage": 220,
+    "warmth": 6,
+    "material_thickness": 6,
+    "use_action": {
+      "type": "holster",
+      "holster_prompt": "Sheath blade",
+      "holster_msg": "You sheath your %s",
+      "min_volume": 1,
+      "max_volume": 10,
+      "multi": 2,
+      "draw_cost": 3,
+      "flags": [ "SHEATH_KNIFE", "SHEATH_SWORD" ]
+    },
+    "flags": [ "BELTED", "WATERPROOF", "ONLY_ONE", "OVERSIZE" ]
+  },
+  {
+    "id": "petpack",
+    "type": "ARMOR",
+    "name": "petpack",
+    "description": "Before the Cataclysm this would allow your four-legged friend to see the world, now it's used to shield them from the world.",
+    "weight": 1316,
+    "volume": 40,
```

Comment:
```suggestion
    "volume": "10 L",
```
Example

Reply:
That might explain why jenkins is freaking out.